invesalius/control.py

```diff
@@ -479,21 +477,21 @@ def Progress(self, data):
                 self.progress_dialog.Close()
                 self.progress_dialog = None
 
-    def OnLoadImportPanel(self, patient_series):
+    def OnLoadImportPanel(self, patient_series: Optional[List]) -> None:
         ok = self.LoadImportPanel(patient_series)
         if ok:
             Publisher.sendMessage("Show import panel")
             Publisher.sendMessage("Show import panel in frame")
             self.img_type = 1
 
-    def OnLoadImportBitmapPanel(self, data):
+    def OnLoadImportBitmapPanel(self, data) -> None:
```

Comment:
Here `data` is a complex type:

`List[Tuple[bytes, str, str, int, int, str, str, wx.WindowIDRef]]`